pkg/logs/input/channel/tailer.go

```diff
@@ -68,3 +66,13 @@ func (t *Tailer) run() {
 		}
 	}
 }
+
+func computeServiceName(lambdaConfig *config.Lambda, serviceName string) string {
+	if lambdaConfig == nil {
```

Comment:
What is this case for?

Reply:
OK, I think I understand. This is in the logs package, and this is the case for the non-Serverless agent, which is 99% of cases.

I think this would be more obvious if this function just took a boolean `isServerless`. We don't really need anything more than that to make a decision.

Reply:
yeah I thought about that but I think this way makes it easier to test. Otherwise, we need to have a piece of untested logic to create the boolean in the run function (which has no tests at all)